### DISCUSSION TITLE
Deleting MQTT entries by NetAlertX?

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [netalertx/NetAlertX#733] [General]
Greetings,
I don´t know if this can´t be done by design, but could NetAlertX be able to delete an antry in the mqtt broker as well when removing a device from the NetAlertX GUI?
I noticed many dead entities from old NetAlertX devices, which I´m deleting within Home Assistant then (or wih the MQTT Explorer)

**Comment by jokob-sk** [comment]
Hey,

Currently, that's unsupported. I might do it in the far future ( a lot of other stuff needs to be done first).

**Reply by AdmiralRaccoon** [reply]
No Problem, thanks!
